lib/_stream_writable.js

```diff
@@ -432,7 +434,7 @@ function clearBuffer(stream, state) {
 }
 
 Writable.prototype._write = function(chunk, encoding, cb) {
-  cb(new Error('not implemented'));
+  cb(new Error('Write method cannot be invoked on the Writable stream . Method not implemented'));
```

Comment:
I think this could be as simple as `_write() must be implemented` or `_write() is not implemented`.